test/apps/iodemo/io_demo.cc

```diff
@@ -2374,6 +2378,14 @@ class DemoClient : public P2pDemoCommon {
                     bytes_completed = server_info.bytes_completed[op];
                     num_completed   = server_info.num_completed[op];
 
+                    if (opts().per_conn_info) {
+                        double mbs = (bytes_completed / (elapsed * UCS_MBYTE));
+                        long iops  = (long)(num_completed / elapsed);
+
+                        conn_log << io_op_names[op] << " " << mbs << "MBs "
+                                 << "iops: " << iops << " | ";
+                    }
```

Comment:
can we move this outside of if/else - unite with lines 2407-2411?

Reply:
done